src/encoded/schemas/genetic_modification.json

```diff
@@ -293,6 +296,9 @@
                                 },
                                 "introduced_elements": {
                                     "enum": ["gRNAs and CRISPR machinery"]
+                                },
+                                "guide_type": {
+                                    "enum": ["sgRNA", "pgRNA"]
```

Comment:
I think if you are allowing any value in the enum (i.e. not a subset) you don't need to specify in the dependancy.  But it doesn't hurt